CMakeLists.txt

```diff
@@ -22,14 +22,12 @@ endif(ANVILL_ENABLE_INSTALL)
 
 add_subdirectory("libraries")
 
-set(_VCPKG_SHARE_DIR "${Z_VCPKG_ROOT_DIR}/installed/${VCPKG_TARGET_TRIPLET}/${CMAKE_INSTALL_DATADIR}")
-
-find_package(XED CONFIG REQUIRED HINTS "${_VCPKG_SHARE_DIR}")
-find_package(gflags CONFIG REQUIRED HINTS "${_VCPKG_SHARE_DIR}")
-find_package(glog CONFIG REQUIRED HINTS "${_VCPKG_SHARE_DIR}")
+find_package(XED CONFIG REQUIRED)
+find_package(gflags CONFIG REQUIRED)
+find_package(glog CONFIG REQUIRED)
 find_package(Z3 4.8 CONFIG REQUIRED)
```

Comment:
I think latest Z3 in vcpkg is past 4.8, and so you might be able to remove this version now too.

Reply:
[ee1d7a1](https://github.com/lifting-bits/anvill/pull/339/commits/ee1d7a18d41ab12a36492b47836f04de93081932)

Yep, VCPKG has 4.11 so this can go.

Reply:
As an aside, version handling in CMake is a bit complicated. It _could_ be used to specify that 4.8 is the minimum compatible version. Still, since I don't think we've tested that and we're not aiming for compatibility outside of vcpkg (at least for now), then removal of this is probably best.

See here for all the details of CMake `find_package` version searching https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-version-selection